css/main.css

```diff
@@ -325,3 +325,118 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.container-footer {
+  padding: 120px 24px;
+}
+
+.text-1 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+form {
+  display: flex;
+  margin: 0 auto;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
```

Comment:
- [ ] There is no space between your paragraph and your field, Please put some space between your paragraph and your field as it is on Figma design 
:upside_down_face:  :upside_down_face: 

See the difference between your Design and the Figma one

![formFigma](https://user-images.githubusercontent.com/88894525/190251719-0b496e42-9059-47d5-869e-abb5f0502c63.png)

![copy](https://user-images.githubusercontent.com/88894525/190251813-d08e50f5-ae45-49f0-ad75-638f9d9497c5.png)